components/aws/sagemaker/tests/integration_tests/resources/definition/groundtruth_pipeline.py

```diff
@@ -0,0 +1,54 @@
+import kfp
+import json
+import copy
+from kfp import components
+from kfp import dsl
+from kfp.aws import use_aws_secret
+
+sagemaker_gt_op = components.load_component_from_file('../../ground_truth/component.yaml')
+
+@dsl.pipeline(
+    name='SageMaker GroundTruth image classification test pipeline',
+    description='SageMaker GroundTruth image classification test pipeline'
+)
+def ground_truth_test(region='',
+    ground_truth_train_job_name='',
+    ground_truth_label_attribute_name='',
+    ground_truth_train_manifest_location='',
+    ground_truth_output_location='',
+    ground_truth_task_type='',
+    ground_truth_worker_type='',
+    ground_truth_label_category_config='',
+    ground_truth_ui_template='',
+    ground_truth_title='',
+    ground_truth_description='',
+    ground_truth_num_workers_per_object='',
+    ground_truth_time_limit='',
+    ground_truth_task_availibility='',
+    ground_truth_max_concurrent_tasks='',
+    role='',
+    workteam_arn=''
+    ):
+
+    ground_truth_train = sagemaker_gt_op(
+        region=region,
+        role=role,
+        job_name=ground_truth_train_job_name,
+        label_attribute_name=ground_truth_label_attribute_name,
+        manifest_location=ground_truth_train_manifest_location,
+        output_location=ground_truth_output_location,
+        task_type=ground_truth_task_type,
+        worker_type=ground_truth_worker_type,
+        workteam_arn=workteam_arn,
+        label_category_config=ground_truth_label_category_config,
+        ui_template=ground_truth_ui_template,
+        title=ground_truth_title,
+        description=ground_truth_description,
+        num_workers_per_object=ground_truth_num_workers_per_object,
+        time_limit=ground_truth_time_limit,
+        task_availibility=ground_truth_task_availibility,
+        max_concurrent_tasks=ground_truth_max_concurrent_tasks
+    ).apply(use_aws_secret('aws-secret', 'AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY'))
```

Comment:
We are now using OIDC.

Reply:
None of the integ tests are using OIDC currently, we can change that in one go in a separate PR.

Reply:
Oh my bad, in my PR I am migrating them to OIDC. I'll rebase on this and change it.